packages/@glimmer/syntax/lib/v1/visitor-keys.ts

```diff
@@ -35,11 +34,6 @@ const visitorKeys = {
   Hash: ['pairs'],
   HashPair: ['value'],
   BlockParam: [],
-
-  // v2 new nodes
-  NamedBlock: ['attributes', 'modifiers', 'children', 'comments'],
-  SimpleElement: ['attributes', 'modifiers', 'children', 'comments'],
-  Component: ['head', 'attributes', 'modifiers', 'children', 'comments'],
```

Comment:
I "ensured these stays in sync with typing" as instructed by the comment on line 3

I don't think these are meant to be here, v2 nodes won't appear in the v1 AST and defining visitors for them doesn't do anything, and in any case, these aren't even in sync with what is currently in the "v2" AST